### PR TITLE
Tk aws infrastructure setup

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: manual/tdr-create-aws-instructure-setup.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+2. Set up individual TDR "environment" AWS accounts to represent the different application environments required. 
+
+   For example setup AWS accounts for "intg", "staging" and "production" environments
+   
+3. Add the TDR "environment" AWS account ids to the parameter store in the TDR "management" AWS account with the following names:
+
+   * Integration Account: /mgmt/intg_account
+   * Staging Account: /mgmt/staging_account
+   * Production Account: /mgmt/prod_account
```
Comment (suzannehamilton) [inline manual/tdr-create-aws-instructure-setup.md]: I can't decide whether it belongs in this section, but the `/mgmt/cost_centre` variable currently needs to be created by hand as well.

File: manual/tdr-create-aws-instructure-setup.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      
+      * Owner: TDR Backend
+      * Terraform: false
+      * Name: TDR Bootstrap Terraform State Lock Table
+   
+      This will provide locking of the s3 bucket to prevent conflicts when multiple developers are working.
+      
+   For further information on setting up Terraform s3 backend, see here: https://www.terraform.io/docs/backends/types/s3.html
+
+2. Setup the Terraform Backend project in the TDR AWS management account: https://github.com/nationalarchives/tdr-terraform-backend/blob/master/README.md
```
Comment (suzannehamilton) [inline manual/tdr-create-aws-instructure-setup.md]: Just to be clear - does "setup" mean, "run the Terraform scripts"? It might be good to rephrase it to make it more obvious that you need to actually run the project to create all the resources, rather than just set up the development environment.

File: manual/tdr-create-aws-instructure-setup.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+       
+      This will store the state of the Terraform for the backend during development, and allow sharing of the state between multiple developers
+   
+   (b) Create DynamoDb table: tdr-bootstrap-terraform-statelock
+   
+      Ensure the DyanamoDb table has the following tags:
+      
+      * Owner: TDR Backend
+      * Terraform: false
+      * Name: TDR Bootstrap Terraform State Lock Table
```
Comment (suzannehamilton) [inline manual/tdr-create-aws-instructure-setup.md]: Could you add the CostCentre tag to steps (a) and (b)? (Though just add a placeholder for the actual value - it's not very sensitive, but we've kept it out of the other public repos.)

File: manual/tdr-create-aws-instructure-setup.md (hunk 1 above)
Comment (TomJKing) [reply manual/tdr-create-aws-instructure-setup.md]: Resolved

File: manual/tdr-create-aws-instructure-setup.md (hunk 2 above)
Comment (TomJKing) [reply manual/tdr-create-aws-instructure-setup.md]: Resolved

File: manual/tdr-create-aws-instructure-setup.md (hunk 3 above)
Comment (TomJKing) [reply manual/tdr-create-aws-instructure-setup.md]: Resolved
